### ISSUE TITLE
Iterative function

### ISSUE THREAD

**Issue by rubiotorres** [yourfirstpr/yourfirstpr.github.io#234]
Can every recursive program become iterative?

**Comment by Sayan-dev** [comment]
Yes, actually you can convert every recursive programs into it's non-recursive part following a set of rules.

**Comment by Nikitapande** [comment]
Yes, recursion gives us behind the scenes can be also be done by implementing a call stack directly in the code itself
